README.md

```diff
@@ -44,11 +44,15 @@ Have ideas for new SDKs or DSLs? See the [Jira](https://issues.apache.org/jira/b
 
 ### Runners
 
-Beam supports executing programs on multiple distributed processing backends. After the Beam project's initial bootstrapping completes, it will include:
-  1. The `DirectPipelineRunner` runs the pipeline on your local machine.
-  2. The `DataflowPipelineRunner` submits the pipeline to the [Google Cloud Dataflow](http://cloud.google.com/dataflow/).
-  3. The `SparkPipelineRunner` runs the pipeline on an Apache Spark cluster. See the code that will be donated at [cloudera/spark-dataflow](https://github.com/cloudera/spark-dataflow).
-  4. The `FlinkPipelineRunner` runs the pipeline on an Apache Flink cluster. See the code that will be donated at [dataArtisans/flink-dataflow](https://github.com/dataArtisans/flink-dataflow).
+Beam supports executing programs on multiple distributed processing backends (runners). It currently includes the following Runners:
```

Comment:
I think of "distributed processing backends" as being different from "runners". A runner is the component that takes a Beam pipeline and submits it to the "processing backend", but doesn't include it. E.g., FlinkPipelineRunner vs. Flink backend.

Suggested: "Beam supports executing programs on multiple distributed processing backends. Currently, it supports the following:"